Procfile

```diff
@@ -1,2 +1,3 @@
 web: bin/upgrade && PYTHONUNBUFFERED=1 bin/web --no-debug --addr=:${PORT}
 worker: bin/upgrade && PYTHONUNBUFFERED=1 bin/worker --no-debug -n 4 -l ${LOG_LEVEL}
+static: node_modules/.bin/webpack -d --watch
```

Comment:
Ok that's what I thought.  I only tried this because Travis was spitting out that it Failed at the freight@0.0.0 postinstall script.  postinstall: `webpack -p`